test/test_helpers/publishing_api_test.rb

```diff
@@ -611,6 +611,30 @@
     end
   end
 
+  describe "#stub_publishing_api_graphql_query" do
+    it "returns the given response" do
+      query = "some query"
+
+      stubbed_response = {
+        edition: {
+          title: "some title",
+        },
+      }
```

Comment:
**suggestion:** realistic response

Should this be nested in a hash under `data`?

Reply:
Yes, it should be.  I'll update that.